openreferee_server/notify.py

```diff
@@ -21,7 +21,7 @@ def log_error(self, *args):
 
     def send(self, payload):
         try:
-            self.session.post(self.url, data=json.dumps({"payload": payload}))
+            self.session.post(self.url, json={"payload": payload})
```

Comment:
Just sondering, do you really need to wrap it in `payload`? Seems to be cleaner to jsonify your payload directly:

```suggestion
            self.session.post(self.url, json=payload)
```

Reply:
It also allows me to "type" the payload, so I can serialise it based on the expected structure associated with the payload.